schema.sql

```diff
@@ -330,6 +330,15 @@ CREATE TABLE IF NOT EXISTS `tile_store` (
   FOREIGN KEY (`house_id`) REFERENCES `houses` (`id`) ON DELETE CASCADE
 ) ENGINE=InnoDB;
 
+CREATE TABLE IF NOT EXISTS `live_casts` (
```

Comment:
Agrees with @DSpeichert . If we are afraid of noobs using their regular password for casting we can enforce own passwords through random string generation. The user can request a password and password change on his cast, but not explicit decide what that password is as its being generated automatically for security reasons.